### DISCUSSION TITLE
Prometheus meta info of k8s

### DISCUSSION THREAD

**Discussion by haiakab** [apache/skywalking#12247] [Q&A]
Hi, I'm using the skywalking to monitor k8s. one of the levels that I need to show is the service level.
in Prometheus I have this label only for kube-stat-metrics and it doesn't have the right value - 
for example : for one of the metrics, _**service="prometheus-kube-state-metrics"**_ which is the exporter itself and the actual service is different, 
for cAdvisor I don't have the service at all 
in both cases, I can extract the service from different labels. (for example for _**pod_kube_info**_ metric  it will be the first part of the pod value, and for _**kube_deployment_spec_replicas**_ it will be the deployment label value)

depending on this I have two questions:
1. how should I configure/install Prometheus/OTEL to get the right service values?
2. in case I need to use different values for service (so let's say if deployment exists it will take it otherwise it will take the pod value (or some regex on it)), is it possible to do so?

**Comment by wu-sheng** [comment]
Try the showcase, https://skywalking.apache.org/docs/skywalking-showcase/next/readme/

**Reply by haiakab** [reply]
I already deployed it, I have the connectivity to Prometheus ( kubernetes is deployed, I'm also able to see all clusters and nodes data in the UI). the issue is with the data itself in Prometheus. I don't have this label so my question that how you have in Prometheus so you didn't need to add any customization in showcase configuration?

**Reply by wu-sheng** [reply]
I am not following. demo.skywalking.apache.org is just deploying this on gke. No custom change.
